### ISSUE TITLE
[BUG]Tags for SqlDataBase is being returned as null even though Tags  are present 

### ISSUE THREAD

**Issue by sumangit789** [Azure/azure-sdk-for-java#10899]
**Describe the bug**
I am retrieving the Tags for SqlDatabase  .But tags are returned as null,even though Tags  are present 

***Exception or Stack Trace***
Add the exception log and stack trace if available
tags...null
**To Reproduce**
1.Get a Sqlserver
2.Get sql database of that server 
3.Get tags 

***Code Snippet***
Below is sample .Not exact code.
List<SqlServer> sqlservers = azure.sqlServers().listAsync();//get all sql server
SqlServer sqlserver = sqlservers.get(0);
//get sqldatabase from server 
SqlDatabase sqlDatabase = azureClient.sqlServers().databases().listBySqlServer(sqlServer);
get tags
Map<String,String> tags = sqlDatabase.inner().getTags();
System.out.println("tags.."+tags).

**Expected behavior**
Tags attached to sqldatabase should be returned.

**Screenshots**
NA
**Setup (please complete the following information):**

os:mac
IDE :Intellij
Version used:1.21.0
 <groupId>com.microsoft.azure</groupId>
            <artifactId>azure</artifactId>
            <version>1.21.0</version>

**Additional context**
NA
**Information Checklist**
Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
- [ ] Bug Description Added
- [ ] Repro Steps Added
- [ ] Setup information Added
NA

**Comment by yungezz** [comment]
hi @xccc-msft could you pls have a look? thanks

**Comment by xccc-msft** [comment]
@sumangit789 Can you please try to capture the request/response and verify if the tags attached in the response?

**Comment by weidongxu-microsoft** [comment]
@haolingdong-msft 

Please verify whether this is fixed in 2.6.0

**Comment by haolingdong-msft** [comment]
The issue has been fixed in azure-resourcemanager 2.6.0
Sample code as below:
```
final String adminUser = "YOUR_USERNAME_HERE";
final String sqlServerName = "YOUR_SERVER_NAME_HERE";
final String sqlDbName = "YOUR_DB_NAME_HERE";
final String dbPassword = "YOUR_PASSWORD_HERE";
final String firewallRuleName = "YOUR_RULE_NAME_HERE";

SqlServer sampleSQLServer = azureResourceManager.sqlServers().define(sqlServerName)
        .withRegion(Region.US_EAST)
        .withNewResourceGroup("sampleSqlResourceGroup")
        .withAdministratorLogin(adminUser)
        .withAdministratorPassword(dbPassword)
        .defineFirewallRule(firewallRuleName)
        .withIpAddressRange("0.0.0.0","255.255.255.255")
        .attach()
        .create();

Map<String, String> tags = new HashMap<>();
tags.put("tag-test-key", "tag-test-value");
SqlDatabase sampleSQLDb = sampleSQLServer.databases().define(sqlDbName).withTags(tags).create();

System.out.println(sampleSQLDb.innerModel().tags()); // output: {tag-test-key=tag-test-value}
```
